src/net/sourceforge/kolmafia/swingui/listener/LicenseDisplayListener.java

```diff
@@ -9,10 +9,8 @@
   private static final String[] LICENSE_FILENAME = {
     "kolmafia-license.txt",
```

Comment:
kolmafia-license.txt is missing, but I think that predates your change.  Should be in src/licenses, but it's no longer there and the page in About is blank.